viz.yaml

```diff
@@ -192,7 +192,7 @@ publish:
     context:
       sections: [ "header-section", "figure-section", "footer-section" ]
       resources: [ "main-css", "svg-css-map", "header-footer-css", "nouislidercss", "noiuipipcss", "jquery",
-        "tooltipsy", "nouislider", "appJS", "mapJS" ]
+        "tooltipsy", "nouislider", "appJS", "mapJS", "<link href=\"https://fonts.googleapis.com/css?family=Open+Sans:400,700\" rel=\"stylesheet\"/>" ]
```

Comment:
I'll work on making this not a hack for the next viz (maybe even back it out into this one).